### ISSUE TITLE
Host these examples somewhere as a publicly available demo

### ISSUE THREAD

**Issue by sohkai** [bigchaindb/bigchaindb-examples#8]
Way lower barrier of entry for most people coming into BigchainDB and these examples.

Once we have a transactions viewer for BigchainDB, it'd also be nice to hook that into these public examples so people can verify what they've done and explore `THE BLOCKCHAIN`.

**Comment by diminator** [comment]
agreed, it would also require us to reset the DB now and then, avoid throttling and other types of misuse
